ui/admin/app/components/form/auth-method/ldap/index.hbs

```diff
@@ -0,0 +1,662 @@
+      }}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  <Hds::Form::Toggle::Field
+    name='anon_group_search'
+    checked={{@model.anon_group_search}}
+    disabled={{form.disabled}}
+    {{on 'change' this.toggleField}}
+    as |F|
+  >
+    <F.Label>{{t
+        'resources.auth-method.form.anon_group_search.label'
+      }}</F.Label>
+    <F.HelperText>{{t
+        'resources.auth-method.form.anon_group_search.help'
+      }}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  <div class='form-section'>
+    <p class='hds-typography-display-300 hds-font-weight-bold'>{{t
+        'resources.auth-method.section.user-entries.title'
+      }}</p>
+    <Hds::Form::HelperText @controlId='helper-text-for-section'>{{t
+        'resources.auth-method.section.user-entries.help'
+      }}</Hds::Form::HelperText>
+  </div>
+
+  <Hds::Form::TextInput::Field
+    @isOptional={{true}}
+    @value={{@model.user_dn}}
+    @isInvalid={{@model.errors.user_dn}}
+    @type='text'
+    name='user_dn'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'user_dn')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.auth-method.form.user_dn.label'}}</F.Label>
+    <F.HelperText>{{t 'resources.auth-method.form.user_dn.help'}}</F.HelperText>
+    {{#if @model.errors.user_dn}}
+      <F.Error as |E|>
+        {{#each @model.errors.user_dn as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  <Hds::Form::TextInput::Field
+    @isOptional={{true}}
+    @value={{@model.user_attr}}
+    @isInvalid={{@model.errors.user_attr}}
+    @type='text'
+    name='user_attr'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'user_attr')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.auth-method.form.user_attr.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.auth-method.form.user_attr.help'
+      }}</F.HelperText>
+    {{#if @model.errors.user_attr}}
+      <F.Error as |E|>
+        {{#each @model.errors.user_attr as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  <Hds::Form::TextInput::Field
+    @isOptional={{true}}
+    @value={{@model.user_filter}}
+    @isInvalid={{@model.errors.user_filter}}
+    @type='text'
+    name='user_filter'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'user_filter')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.auth-method.form.user_filter.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.auth-method.form.user_filter.help'
+      }}</F.HelperText>
+    {{#if @model.errors.user_filter}}
+      <F.Error as |E|>
+        {{#each @model.errors.user_filter as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  <Hds::Form::Fieldset @isOptional={{true}} as |F|>
```

Comment:
isn't this same as `<Form::Field::KeyValueList>`?

Reply:
Yeah we should update it to support more than just text fields since it looks like it's getting duplicated because of the usage of a dropdown and text field as the `Value` field

Reply:
Yup I agree, I can add a ticket to refactor `<Form::Field::KeyValueList>` and update ldap and oidc forms to use it.